api/core/app/entities/queue_entities.py

```diff
@@ -326,6 +327,13 @@ class QueueWorkflowStartedEvent(AppQueueEvent):
     graph_runtime_state: GraphRuntimeState
 
 
+class QueueWorkflowSuspendedEvent(AppQueueEvent):
+    event: QueueEvent = QueueEvent.WORKFLOW_SUSPENDED
+    # next_node_id records the next node to execute after resuming
+    # workflow.
+    next_node_id: str
```

Comment:
There may be multiple nodes in the future. Should we consider recording the ID of the previous (or current) node?